Cod3rsGrowth.forms/FormEditarVenda.cs

```diff
@@ -87,39 +74,36 @@ private void CarregarComboBoxCarro()
         {
             try
             {
-                IEnumerable<Carro> obter;
-                var carros = _servico.ObterTodos(_filtro);
+                carros = _servico.ObterTodos(_filtro);
                 var venda = _servicoVenda.ObterTodos(_filtroVenda);
 
                 foreach( var vendas in venda)
```

Comment:
var vendas in venda?
Deveria ser (var venda in vendas)
Refatora esse método, tem forma melhor de fazer sem precisar percorrer tantas listas